docs/sources/menu.yaml

```diff
@@ -204,12 +204,16 @@
 - name: Alerting
   link: /alerting/
   children:
-    - link: /alerting/rules/
-      name: Engine and Rules
-    - link: /alerting/metrics/
-      name: Metrics
+    - link: /alerting/alerts-overview/
+      name: Overview
     - link: /alerting/notifications/
-      name: Notifications
+      name: Alert notifications
+    - link: /alerting/create-alerts/
+      name: Create alerts
+    - link: /alerting/view-alerts/
+      name: View alerts
+    - link: /alerting/troubleshoot-alerts/
```

Comment:
Pause an alert page should be included in alerting menu, right?

Reply:
You can tell I was getting tired, lol. Good catch.